dags/bqetl_download_funnel_attribution.py

```diff
@@ -66,6 +80,10 @@
         pool="DATA_ENG_EXTERNALTASKSENSOR",
     )
 
+    checks__ga_derived__downloads_with_attribution__v2.set_upstream(
+        wait_for_ga_derived__www_site_empty_check__v1
+    )
+
```

Comment:
mmh, I don't think this is correct? There should only be one upstream dependency which is `ga_derived__downloads_with_attribution__v2`